### PR TITLE
add check markdown github action workflow for fix README.md markdown file

### PR DESCRIPTION
**Pull request by samsonasik**
@TomasVotruba I added `.github/inactive_workflows/check_markdown.yml` to fix README.md markdown file on push to master.

It requires `"symplify/easy-coding-standard": "dev-master"` as current addition `check-markdown` command not tagged yet. 

It uses 

```
    "minimum-stability": "dev",
    "prefer-stable": true
```

to allow install it.

### CODE REVIEW

File: .github/inactive_workflows/check_markdown.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+            -   uses: stefanzweifel/git-auto-commit-action@v4.1.6
+                with:
+                    commit_message: Fix php code inside markdown file
+                    branch: "master"
+                    commit_user_name: TomasVotruba
+                    commit_user_email: tomas.vot@gmail.com
+                    commit_author: TomasVotruba <tomas.vot@gmail.com>
+                env:
+                    GITHUB_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```
Comment (TomasVotruba) [inline .github/inactive_workflows/check_markdown.yml:30]: :clap: You're a mind reader

File: composer.json [hunk 2]
```diff
@@ -301,5 +301,7 @@
         "branch-alias": {
             "dev-master": "0.8-dev"
         }
-    }
+    },
+    "minimum-stability": "dev",
+    "prefer-stable": true
```
Comment (TomasVotruba) [inline composer.json:306]: I've tagged the Symplify https://github.com/symplify/symplify/releases/tag/8.2.24 with both ECS commands you made, so you can use it

Comment (samsonasik) [reply composer.json]: updated.

**Comment by samsonasik** [comment]
I've updated using tagged symplify/easy-coding-standard and moved check_markdown.yml from inactive to active workflow

**Comment by TomasVotruba** [comment]
Thank you, let's try it out :+1:

**Comment by samsonasik** [comment]
Got the following error : 

```
INPUT_REPOSITORY value: .
INPUT_BRANCH value: master
Already on 'master'
M	README.md
Your branch is up to date with 'origin/master'.
INPUT_FILE_PATTERN: .
INPUT_COMMIT_OPTIONS: 
[master a9fee85] Fix php code inside markdown file
 1 file changed, 5 insertions(+)
INPUT_TAGGING_MESSAGE: 
 No tagging message supplied. No tag will be added.
remote: error: GH006: Protected branch update failed for refs/heads/master.        
remote: error: 10 of 10 required status checks are expected.        
To https://github.com/rectorphp/rector
 ! [remote rejected] HEAD -> master (protected branch hook declined)
error: failed to push some refs to 'https://github.com/rectorphp/rector'
Error: Invalid status code: 1
    at ChildProcess.<anonymous> (/home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4.1.6/index.js:17:19)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5) {
  code: 1
}
Error: Invalid status code: 1
    at ChildProcess.<anonymous> (/home/runner/work/_actions/stefanzweifel/git-auto-commit-action/v4.1.6/index.js:17:19)
    at ChildProcess.emit (events.js:210:5)
    at maybeClose (internal/child_process.js:1021:16)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:283:5)
```

**Comment by TomasVotruba** [comment]
Me too. Any idea what needs to be fixed?

**Comment by samsonasik** [comment]
probably need `changes_detected` option https://github.com/stefanzweifel/git-auto-commit-action#outputs ,

**Comment by TomasVotruba** [comment]
This workflow should run only on `pull_request`, as `master` is protected from direct commits.

**Comment by samsonasik** [comment]
ok, I created PR https://github.com/rectorphp/rector/pull/4144

**Pull request merged**